tests/tests_pytorch/trainer/logging_/test_logger_connector.py

```diff
@@ -644,3 +644,23 @@ def test_result_collection_no_batch_size_extraction():
     assert results["training_step.epoch_log_val"].value == log_val * batch_size
     assert results["training_step.epoch_log_val"].cumulated_batch_size == batch_size
     assert results["training_step.epoch_sum_log_val"].value == log_val
+
+
+def test_sync_dist_with_torchmetrics():
+    acc = Accuracy(task="multiclass", num_classes=4, top_k=1)
```

Comment:
This is failing in CI when the minimum torchmetrics version is installed. cc @Borda